step-7-test-the-api.md

```diff
@@ -65,7 +65,7 @@ public class RatingsControllerTest extends AbstractIntegrationTest {
                 .when()
                 .post("/ratings")
                 .then()
-                .statusLine("HTTP/1.1 400 Bad Request");
+                .statusCode(400);
```

Comment:
I'm sure that `statusLine` was working a couple of weeks ago, but either way, the service is not returning a status reason code any more. We're getting back, e.g. `HTTP/1.1 202 ` (note amazingly ugly trailing space!)

[This doesn't seem to be a bug](https://github.com/spring-projects/spring-boot/issues/6789), so let's just make the tests match the actual behaviour.